cli/commands/play.go

```diff
@@ -522,34 +524,77 @@ func buildSnakesFromOptions() map[string]SnakeState {
 func printMap(state *rules.BoardState, snakeStates map[string]SnakeState, gameTurn int32) {
 	var o bytes.Buffer
 	o.WriteString(fmt.Sprintf("Ruleset: %s, Seed: %d, Turn: %v\n", GameType, Seed, gameTurn))
-	board := make([][]rune, state.Width)
+	board := make([][]string, state.Width)
 	for i := range board {
-		board[i] = make([]rune, state.Height)
+		board[i] = make([]string, state.Height)
 	}
 	for y := int32(0); y < state.Height; y++ {
 		for x := int32(0); x < state.Width; x++ {
-			board[x][y] = '◦'
+			if UseColor {
+				board[x][y] = "\033[38;2;200;200;200m□"
+			} else {
+				board[x][y] = "◦"
+			}
 		}
 	}
 	for _, oob := range state.Hazards {
-		board[oob.X][oob.Y] = '░'
+		if UseColor {
+			board[oob.X][oob.Y] = "\033[48;2;127;127;127m \033[107m"
```

Comment:
Can you convert these terminal escape codes into a few documented constants? If we want to adjust them later it'll be a lot easier if this looks something like:
```
board[oob.x][oob.y] = TERM_BG_GREY + " " + TERM_BG_WHITE
```
(I'm not sure I've even translated the codes above correctly - another reason to do this 😄 )

Reply:
yea that sounds like a good idea. I am not very well versed in go's idioms. Where would i define does?

Reply:
You can do something like this, maybe in a `colors.go` in the same directory:
https://github.com/BattlesnakeOfficial/rules/blob/31faba642ccb088b62422926ba6327801de2c899/ruleset.go#L7-L11